### PR TITLE
Use janet

### PR DESCRIPTION
**Pull request by jonathanadler**

### CODE REVIEW

File: janet.json [hunk 1]
```diff
@@ -5,6 +5,6 @@
     "*"
   ],
   "exposure": [
-    "wix.com"
+    "bo"
```
Comment (shlomitc) [inline janet.json:8]: Why not wix.com?

**Review by shlomitc — CHANGES_REQUESTED** [review]

File: janet.json (hunk 1 above)
Comment (jonathanadler) [reply janet.json:8]: It didn't work at the beginning, so I tried to change to bo.
But this also doesn't work 🤷‍♂️

**Review by shlomitc — COMMENTED** [review]
It's still failing
`[npm run pr-postbuild --if-present] ⛔️  Deployment failed to finish within 300000 ms`

**Comment by jonathanadler** [comment]
> It's still failing
> `[npm run pr-postbuild --if-present] ⛔️ Deployment failed to finish within 300000 ms`

Yeah, there's no surprise
We'll need an answer from Roy

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-ui-tpa.pr_383/

**Pull request merged**